apps.yaml

```diff
@@ -1,4 +1,18 @@
 ---
+diffusion-electrolyte-mixture:
+    metadata:
+        title: Diffusion_Electrolyte_Mixture
+        description: |
+            This workflow uses the SimStack framework features to perform Molecular Dynamics simulation to calculate the diffusion coefficient of the components in binary mixture of electrolytes.
+        authors: Saientan Bag
+        external_url: https://github.com/saientan/Diffusion_Electrolyte_Mixture
+        documentation_url: https://github.com/saientan/Diffusion_Electrolyte_Mixture/blob/main/README.md
+        logo: https://github.com/saientan/Diffusion_Electrolyte_Mixture/blob/main/Diffusion.png
+        state: development
+        version: '1.0'
+    categories:
```

Comment:
Could you add a WP category? See `categories.yaml` for the list of existing WP categories.

Reply:
Category is already Mentioned?  -classical and technology-simstack

Reply:
@saientan We are specifically asking whether this app can be categorized into one of the **WP** categories.

Reply:
@csadorf Its difficult to categorize the app in one of the WP categories. The app bunch together a complicated simulation workflow. But this is not multi-scale.